velox/dwio/common/tests/utils/E2EFilterTestBase.cpp

```diff
@@ -94,6 +94,7 @@ void E2EFilterTestBase::readWithoutFilter(
     const std::vector<RowVectorPtr>& batches,
     uint64_t& time) {
   dwio::common::ReaderOptions readerOpts{leafPool_.get()};
+  readerOpts.setFileFormat(FileFormat::DWRF);
```

Comment:
Thanks, good catch! The tests in Parquet's `E2EFilterTest.cpp` are all successful because this parameter is not used. I will modify it in the subsequent PR.